tests/image_test.py

```diff
@@ -6,7 +6,7 @@ def test_is_not_default_image():
     int_path = os.path.normpath(os.path.join(__file__, '..', 'test_files', 'integration-Zoom.yml'))
     image.INTEGRATION_REGEX = int_path
     image_validator = image.ImageValidator(int_path)
-    assert image_validator.is_not_default_image() is True
+    assert image_validator.is_not_default_image() is False
```

Comment:
Why?

Reply:
is_not_default_image is just a string comparison, and we check for the base64 string representation of an image, so we don't need to check python's string comparison correctness, So I've changed the image in that integration to be also the default one so that we can use the same file also for other tests :)